src/fxos/read_gaia_commit.js

```diff
@@ -0,0 +1,23 @@
+let adb = require('../adb');
+let debug = console.log.bind(console, '[fxos/read_gaia_commit]');
+let denodeify = require('denodeify');
+let exec = require('mz/child_process').exec;
+let readFile = require('mz/fs').readFile;
+let tmpdir = denodeify(require('tmp').dir);
+
+module.exports = async function readGaiaCommit() {
+  let local = await tmpdir();
+  debug('tmpdir', local);
+  await adb.pull(
+    '/system/b2g/webapps/settings.gaiamobile.org/application.zip',
```

Comment:
From what I understand, the application.zip for settings can be in one of two different places:

* /system/b2g/webapps/settings.gaiamobile.org
* /data/local/webapps/settings.gaiamobile.org